hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/handlers/ContractCreateHandler.java

```diff
@@ -84,7 +82,6 @@ public void handle(@NonNull final HandleContext context) throws HandleException
     public void pureChecks(@NonNull final TransactionBody txn) throws PreCheckException {
         final var op = txn.contractCreateInstanceOrThrow();
 
-        // FUTURE:Consider reading the init code from a file but this may be too much to do for pure checks
         final var intrinsicGas = gasCalculator.transactionIntrinsicGasCost(Bytes.wrap(new byte[0]), true);
```

Comment:
(I guess the evaluation is: In fact it wasn't worth it to do.)

Reply:
yes, too expensive to do here